Emby.Server.Implementations/LiveTv/TunerHosts/M3uParser.cs

```diff
@@ -168,28 +168,24 @@ private string GetChannelNumber(string extInf, Dictionary<string, string> attrib
             string numberString = null;
             string attributeValue;
 
-            if (attributes.TryGetValue("tvg-chno", out attributeValue))
+            if (attributes.TryGetValue("tvg-chno", out attributeValue)
+                && double.TryParse(attributeValue, CultureInfo.InvariantCulture, out _))
             {
-                if (double.TryParse(attributeValue, NumberStyles.Any, CultureInfo.InvariantCulture, out _))
-                {
-                    numberString = attributeValue;
-                }
+                numberString = attributeValue;
             }
 
             if (!IsValidChannelNumber(numberString))
             {
                 if (attributes.TryGetValue("tvg-id", out attributeValue))
                 {
-                    if (double.TryParse(attributeValue, NumberStyles.Any, CultureInfo.InvariantCulture, out _))
+                    if (double.TryParse(attributeValue, CultureInfo.InvariantCulture, out _))
                     {
                         numberString = attributeValue;
                     }
-                    else if (attributes.TryGetValue("channel-id", out attributeValue))
+                    else if (attributes.TryGetValue("channel-id", out attributeValue)
+                        && double.TryParse(attributeValue, CultureInfo.InvariantCulture, out _))
```

Comment:
identation

Reply:
What's wrong with it?

Reply:
I thought it was supposed to be indented like this?:

```
    attributes.TryGetValue("channel-id", out attributeValue)
    && double.TryParse(attributeValue, CultureInfo.InvariantCulture, out _)
```

Reply:
Not afaik, I've always just indented one level, it just happens to line up with `if (`
Looking trough the code now I did find multiple places where you did this :p